Tests/TestBundles/DefaultTestBundle/Entity/BasicConstraintsEntity.php

```diff
@@ -711,7 +711,7 @@ public function isValidSingleChoice()
      */
     public function isValidMultipleChoice()
     {
-        return $this->_t_get(array('June', 'July'), array('June', 'May', 'September'));
+        return $this->_t_get(array('June', 'July'), array('June', 'May'));
```

Comment:
Symfony stopped showing both error messages since 2.5.5. That's described in https://github.com/formapro/JsFormValidatorBundle/pull/83

So i kept only one invalid option and changed the last supported Symfony version of this branch to <2.7. Otherwise we could not support Symfony from 2.5.5 to <2.7 so easily.